docs/runbook/Glossary.md

```diff
@@ -1,276 +0,0 @@
-# Glossary
```

Comment:
Why is this gone?

Reply:
I believe the useful part of the glossary is a repeat of what I'd found on the main Terragraph doc page. I link to that instead in the document intro section.

If there are terms that you feel that glossary does not have but the one I deleted did, let me know. I might modify this file instead of deleting it.

Reply:
Ok. I will list the terms that I think are potentially important to keep that are not in the TG glossary. I'll leave it to you to decide how important they are. Also, they don't necessarily need to be in a glossary, they can be defined upon first use (as you mentioned earlier).

AOI
Candidate Graph
CAPEX
CIR
CN
DHM
DSM
DTM
Nb times on MCS route
OPEX
OSM

Reply:
Thanks!

AOI - defined in Input Files / Boundary Polygon File
Candidate Graph - candidate topology file is used throughout Input Files and candidate topology is defined in the Output Files; we don't use the term candidate graph much. I could see keeping it around but usually it seems clear from context.
Capex - common terminology, can be googled
CIR - not really used; we tend to use the term demand rather than CIR
CN - in the other doc
DHM/DSM/DTM - defined in input files
Nb times on MCS route - no longer needed (used to be in the output data)
Opex - no longer use this
OSM - not used

My gut is that a separate glossary is not needed for these terms.